docs/STORAGE.md

```diff
@@ -0,0 +1,21 @@
+# Storage
+
+We are using the Google Storage Jenkins plugins but it's disable for the time being since it got some issues with some parallelisation.
```

Comment:
```suggestion
We are using the Google Storage Jenkins plugins but it is disabled for the time being because it has some issues with some parallelisation.
```